Sources/SwiftDocCUtilities/Action/Actions/Init/CatalogTemplateKind.swift

```diff
@@ -0,0 +1,352 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2023 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import SwiftDocC
+
+/// Specifies the different template kinds available for
+/// initializing the documentation catalog.
+public enum CatalogTemplateKind: String {
+    
+    /// A template designed for authoring conceptual documentation consisting of a catalog containing only articles.
+    /// This template contains an expanded structure and content with guidelines on how to author content using DocC.
+    case expanded
+    
+    /// A template designed for authoring conceptual documentation consisting of a catalog containing only one markdown file.
+    /// This template contains minimal content to let the user start writing content right away.
+    case minimal
```

Comment:
These are back to the same problem again. It's important for user experience to make something that's not only easy to use right but that's hard to use wrong. These names which are surfaced in the CLI makes it too easy for someone to create a catalog for their framework—thinking that it's the right way to create a minimal starting catalog. 

Even _if_ someone who is new do DocC reads the help text carefully I wouldn't expect them to understand from this description that they should only use this for articles and that this template is not suitable for symbol documentation.

> This template is crafted for creating conceptual documentation and includes a catalog with just one markdown file.

Reply:
I'm still of the opinion that we shouldn't have article only templates at all. Instead we should fix the requirement to explicitly define a technology root so that developers can just put one or more markdown files in a directory with a '.docc' extension and pass that to DocC. Once that is fixed the "minimal" template is too small IMO and the "expanded" template is better served as documentation.

Reply:
I'm also not convinced that the tutorial template is the _best_ solution to making it easier to get started writing tutorial content. I raised both of these concerns in the [forum thread][1] with more details there. 

[1]: https://forums.swift.org/t/pitch-docc-init-command-catalog-templates/67648/4 

I don't know why we're moving ahead with the same solution without addressing any of the concerns about this solution and a closer look at the specific problems it's trying to solve.